### PR TITLE
Fix multi gpu not on separate lines

### PR DESCRIPTION
**Pull request by HO-COOH**
This is how `neofetch` showing multiple gpus
image

This is how `winfetch` showing multiple gpus before this change
image

This is after this change
image

### CODE REVIEW

**Comment by rashil2000** [comment]
Please do not make formatting changes in the PR. Only change the working code.

**Comment by HO-COOH** [comment]
Okay. Didn't notice vscode reformat something.

**Comment by rashil2000** [comment]
Winfetch already supports multiline output. You should modify the `info_gpu` function to handle multiple GPUs better. See the `info_colorbar` to see how it's done.

What is the output of `(Get-CimInstance -ClassName Win32_VideoController -Property Name).Name` on your system?

**Comment by rashil2000** [comment]
There are still some whitespace changes in the bottom part, kindly revert them

File: winfetch.ps1 [hunk 1]
```diff
@@ -1091,4 +1096,4 @@ if (-not $stripansi) {
 # | __/ _ \| __|
 # | _| (_) | _|
 # |___\___/|_|
-#
+#
```
Comment (rashil2000) [inline winfetch.ps1:1099]:
```suggestion
#

```

File: winfetch.ps1 [hunk 2]
```diff
@@ -703,10 +703,15 @@ function info_cpu {
 }
 
 function info_gpu {
-    return @{
+    [System.Collections.ArrayList]$lines = @()
+    #loop through Win32_VideoController
+    foreach ($gpu in Get-CimInstance -ClassName Win32_VideoController -CimSession $cimSession) {
```
Comment (rashil2000) [inline winfetch.ps1:708]:
```suggestion
    foreach ($gpu in Get-CimInstance -ClassName Win32_VideoController -Property Name -CimSession $cimSession) {
```

File: winfetch.ps1 [hunk 3]
```diff
@@ -703,10 +703,15 @@ function info_cpu {
...
+        [void]$lines.Add(@{
         title   = "GPU"
```
Comment (rashil2000) [inline winfetch.ps1:710]:
```suggestion
            title   = "GPU"
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Pull request merged**